src/neptune_scale/utils.py

```diff
@@ -0,0 +1,42 @@
+from collections.abc import Collection
+from datetime import datetime
+from typing import Any
+
+
+def stringify_unsupported(d: dict[str, Any], **kwargs: Any) -> dict[str, Any]:
+    """
+    A helper function that flattens a nested dictionary structure and casts unsupported values to strings to be logged in Neptune.
+    Note:
+    - Sequence and Set collections (like list, tuple, set, etc.) are converted to strings.
+    - None values are ignored.
+
+    Args:
+        d: Dictionary to flatten
+        **kwargs: Additional arguments for backward compatibility
+
+    Returns:
+        dict: Flattened dictionary with string keys and cast values
+
+    For more details, see https://docs.neptune.ai/api/utils/#stringify_unsupported
```

Comment:
This link doesn't work

Reply:
The docs team will publish this page shortly before/after the release. URL can be updated before release based on docs feedback